src/Common/Core/Twig/Extensions/TwigFilters.php

```diff
@@ -109,4 +109,20 @@ public static function getFilters(&$twig, $app)
         $twig->addFilter(new Twig_SimpleFilter('timeago', $app.'::timeAgo'));
         $twig->addFilter(new Twig_SimpleFilter('formatdatetime', $app.'::formatDateTime'));
     }
+
+    /**
+     * Show deprecated error for sprintf.
+     *
+     * @return string
+     */
+    public static function deprecatedSprintf()
+    {
+        trigger_error(
+            '|sprintf is deprecated.
+             Use |format(args)|raw instead',
```

Comment:
our implementation of sprintf did the |raw automatically
Ill make it more clear that the |raw is only needed if you want to include html